### DISCUSSION TITLE
I can't connect and verify my mail connect to gitcoin. Please help?

### DISCUSSION THREAD

**Discussion by thuongnexo** [community/community#37685] [New to GitHub] [answered]
I tried so many way to connect my gitcoin to email. But it connect too long and never success. Please help

**Comment by jfullstackdev** [comment, marked as answer]
fortunately, it's hosted on GitHub too

https://github.com/gitcoinco

you can ask them there and hopefully they will respond

**Reply by vikrant0705** [reply]
Hlo sir good afternoon
